src/OSSupport/File.cpp

```diff
@@ -453,6 +453,29 @@ AString cFile::ReadWholeFile(const AString & a_FileName)
 
 
 
+AString cFile::ReplaceFileNameInvalidChars(const AString & a_FileName, char a_Replacement)
+{
+	AString res(a_FileName);
+	for (auto & ch: res)
+	{
+		switch (ch)
+		{
+			case ':':
+			case '\\':
```

Comment:
It is possible to create a file on linux that contains `\` or `:`. I think removing invalid chars is the wrong approach.